File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        long long y = x;
        long long rev = 0;
        while(x > 0){
            
            rev = rev * 10 + x % 10;
            x/=10;
        }
        if (rev == y){
            return true;
        }
        else{
            return false;
        }
    }
    
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        a = x
        rev = 0
        while (x > 0):
            rev = rev * 10 + x % 10
            x = x//10
        if rev == a:
            return True
        else:
            return False